settings/common.py

```diff
@@ -459,6 +459,16 @@
     },
 }
 
+CELERY_ROUTES = {
+    re.compile(r"(checks)\.tasks\..*"): {"queue": "rule-check"},
+    re.compile(
+        r"(exporter|importer|notifications|publishing|workbaskets)\.tasks\..*",
```

Comment:
As there are only two routes, each being the complement of the other, could this be expressed as a _not checks_ regex. Something like
```suggestion
        r"(?!checks)\.tasks\..*",
```